pos_order_reorder/__manifest__.py

```diff
@@ -6,7 +6,6 @@
     "depends": ["point_of_sale"],
     "website": "https://github.com/OCA/pos",
     "author": "Cetmix,Odoo Community Association (OCA)",
-    "maintainers": ["GabbasovDinar", "CetmixGitDrone"],
```

Comment:
I don't get it.

https://github.com/GabbasovDinar doesn't seems to be a bot / shared account.

Reply:
> @legalsylvain are you 100% sure he is not a bot?

No, but I have confidence. 

- https://github.com/GabbasovDinar looks human.
- https://github.com/CetmixGitDrone seems to be a bot.

It's enough for me.